### PR TITLE
IssueCommand/PrCommand: Use embeds; allow querying by issue/PR number

### PR DESCRIPTION
**Pull request by mattco98**
Also uses the actual emote for `:^)`

### CODE REVIEW

File: src/commands/prCommand.ts [hunk 1]
```diff
@@ -17,11 +19,69 @@ export class PRCommand implements Command {
...
+                await parsedUserCommand.send(embed);
+
+                return;
+            }
+        }
+
         const result = await githubAPI.search_pull_requests(parsedUserCommand.args.join("+"));
         if (result) {
-            await parsedUserCommand.send(`${result.html_url}`);
+            const pr = await githubAPI.search_pull_request(result.number);
```
Comment (mattco98) [inline src/commands/prCommand.ts:44]: I don't fully understand why I need to do a second request here with the PR number to get the proper type, but I don't have to do that with the issues. I might be doing something wrong, maybe with octokit?

**Comment by bgianfo** [comment]
Do we actually need the Numeric search specializations? The text versions work for both scenarios in my experience?

File: src/commands/prCommand.ts (hunk 1 above)
Comment (bgianfo) [reply src/commands/prCommand.ts:44]: Is it because you are missing the `is:pr` and `is:issue` qualifiers?

Comment (bgianfo) [reply src/commands/prCommand.ts:44]: The note here might be related? https://octokit.github.io/rest.js/v18#issues-get

**Comment by bgianfo** [comment]
Looks good enough to merge, I guess we can try to figure out the double request thing later.

**Pull request merged**